plugins/Monolog/Processor/ExceptionToTextProcessor.php

```diff
@@ -85,6 +85,10 @@ private function getStackTrace($exception)
 
     public static function getWholeBacktrace(\Exception $exception, $shouldPrintBacktrace = true)
     {
+        if (!$shouldPrintBacktrace) {
```

Comment:
@diosmosis should this one still be needed?

Reply:
Yes, otherwise the backtrace is always printed. This was an unrelated bug I found.